content/huaweicloud-cpi.md

```diff
@@ -0,0 +1,142 @@
+This topic describes cloud properties for different resources created by the HuaweiCloud CPI.
+The Huaweicloud CPI is designed for user most familiar with the Huawei cloud concepts and it only used in [Huaweicloud](https://www.huaweicloud.com/).
+
+## AZs {: #azs }
+
+Schema for `cloud_properties` section:
+
+* **availability_zone** [String, required]: Availability zone to use for creating instances. Example: `cn-north-1b`.
+
+Example:
+
+```yaml
+azs:
+- name: z1
+  cloud_properties:
+    availability_zone: cn-north-1b
+```
+
+---
+## Networks {: #networks }
+
+Schema for `cloud_properties` section used by dynamic network or manual network subnet:
+
+* **subnet_id** [String, required]: Subnet ID belong to VPC and  the instance will be created.
```

Comment:
We found this sentence to be slightly unclear, do you mean more like something along the lines of: "Subnet ID in which the instances will be created"   ?